package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "covid19-betacode-frontend",
-  "version": "1.5.2",
+  "version": "1.6.2",
```

Comment:
The version should be 1.6.0. Every minor should reset the patch. Every major should reset the minor.